src/swapnaWJun20/SumExceptionHandling.java

```diff
@@ -0,0 +1,37 @@
+package swapnaWJun20;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class SumExceptionHandling {
+
+	int getSumThreeNos() {
+		int numbers = 0;
+		int sum = 0;
+		int count = 0;
+	
+		Scanner sc = new Scanner(System.in);
+		System.out.println("How many numbers you want to add : ");
+		int totalNo = sc.nextInt();	
+		while (count < totalNo) {
+			try {
+				System.out.println("Enter the number to add : ");
+				numbers = sc.nextInt();
+				sum += numbers;
+				count++;
+			} catch (InputMismatchException ie) { //
+				sc.next();
```

Comment:
If the user add a String with space then this can't handle this. So sc.nextLine() is advisable. Also a message for the user would be good.

Reply:
Said changes incorporated. Thanks for comment.